src/couch_index_server.erl

```diff
@@ -38,7 +38,6 @@
 -record(st, {root_dir}).
 
 start_link() ->
-    couch_epi:register_service(couch_index),
```

Comment:
Also since I'm implementing the approach you've proposed (when we pass children to couch_epi:register_service). I would need to add a supervisor.